.github/workflows/main.yml

```diff
@@ -0,0 +1,56 @@
+name: Release to dockerhub
+on:
+  push:
+    tags:
+      - dockerhub_release_*  # Launch dockerhub push on dockerhub_release_* tags
+jobs:
+  build_docker_image:
+    runs-on: ubuntu-latest
+    env:
+      registry: osrf/subt-virtual-testbed
+    strategy:
+      matrix:
+        image_name: ['cloudsim_sim', 'cloudsim_bridge', 'subt_shell', 'subt_sim_entry']
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install base dependencies
+        run: |
+          sudo apt update;
+          sudo apt-get install -y docker
+
+      - name: Run build script
+        run: |
+            cd docker
+            bash +xe ./build.bash ${{ matrix.image_name }}
+
+      - name: Tag image
+        run: docker tag ${{ matrix.image_name }}:latest ${{ env.registry }}:${{ matrix.image_name }}_latest
+
+      # only reserved for subt_sim_entry
+      - name: Tag latest image
+        run: docker tag ${{ matrix.image_name }}:latest ${{ env.registry }}/subt:latest
+        if: matrix.image_name == 'subt_sim_entry'
+
+      - name: Login to docker hub
+        uses: actions-hub/docker/login@master
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Push to docker
+        uses: actions-hub/docker@master
+        env:
+          NAME: ${{ matrix.image_name }}
```

Comment:
We convinced ourselves that this parameter isn't used, right?

Reply:
ouch, forgot that one. thanks